src/routes/profile.js

```diff
@@ -155,47 +198,176 @@ router.get(
   },
 );
 
+const storage = multer.memoryStorage();
+const upload = function (req, res, next) {
+  multer({
+    storage: storage,
+    limits: { fileSize: MAX_FILESIZE },
+  }).fields([
+    { name: 'avatarRes', maxCount: 1 },
+    { name: 'avatarSpy', maxCount: 1 }, // Whitelist, other files will not be accepted
+  ])(req, res, function (err) {
+    // TODO-kev: Check below if it is handling it correctly
+    if (err instanceof multer.MulterError && err.message === 'File too large') {
+      // TODO-kev: Function for 1MB? Enum object?
+      req.flash('error', 'File size exceeds the limit: 1MB.');
```

Comment:
Add a rule in the change avatar page to let users know of the max file size limit.

Reply:
Update the rules in general and the mod avatar approve page.